### DISCUSSION TITLE
iOS AR not opening due to Google Analytics. Anyone else experienced this?

### DISCUSSION THREAD

**Discussion by cpiggott** [google/model-viewer#4797] [Q&A]
Recently discovered that a model-viewer deployment stopped working on iOS for a site. 

After using the Safari iOS debugger and debugging the code, it gets to the point that it is trying to open AR on iOS. I see the console log `Attempting to present in AR with QuickLook...` so I know it's getting to the correct intent URL on iPhone. I can see the intent URL that is generated, and copy+pasting the link into the browser opens the AR session in QuickLook.

When I step through the debugger past the log and get to the click `https://googletagmanager.com/gtm.js` ends up capturing the link click, sending off analytics events(This is an assumption as the gtm.js code is obfuscated) and then finally clicking on the intended link. 

My current assumption is that when the analytics script takes over the click, the user gesture and therefore, the `user intent` is removed, which iOS is then blocking the AR/QuickLook request.

Has anyone else experienced this before?

**Comment by elalish** [comment]
Interesting - so this is analytics that you've added to your website, outside of MV, correct? Because we have no analytics. 

We've seen some recent iOS regressions around the AR link click that appear to be overzealous security fixes, so this may be related. @AdaRoseCannon do you have any ideas here?

**Reply by AdaRoseCannon** [reply]
I'll ask around and see if anyone knows something that could be the cause.

**Reply by cpiggott** [reply]
Yeah, the analytics are on the website but we don't have control over that piece of the site. This issue only happens on iOS, so I'm not sure if it's the analytics firing before the link being clicked or if it's something specific to iOS(We haven't seen this issue anywhere else other than this site).
